js-client-examples/browser-example/package.json

```diff
@@ -51,8 +50,7 @@
         ]
     },
     "devDependencies": {
-        "@fluencelabs/cli": "0.4.20",
-        "@fluencelabs/aqua-lib": "0.6.0",
+        "@fluencelabs/cli": "0.11.4",
```

Comment:
I imagined part of this process would be to show how cli is used together with everything. And here it's still just a part of npm script without fluence.yaml
I don't know if this should be changed or not, since I recognize the simplicity of this part for new-comers

Reply:
These examples use cli just to compile aqua. I think that` fluence.yaml` would be distracting. We have many other docs which include `fluence.yaml` and explain it purpose.